lib/views/subscriptions-scene.js

```diff
@@ -163,11 +164,11 @@ class SubscriptionsScene extends React.Component {
 
     // persist it
     api.post('subscriptions', {
-        channel_id,
-        notifications_enabled: true
-      })
+      channel_id,
+      notifications_enabled: true,
+    })
```

Comment:
TBH I can't justify enforcing semicolons. I like them personally but can't think of any reason to enforce them apart from consistency.
I can revert that change if you want.